tests/test_leadtime.py

```diff
@@ -0,0 +1,24 @@
+from chiltepin.jedi.leadtime import leadtime_to_seconds, seconds_to_leadtime
+
+def test_leadtime_to_seconds():
+    assert leadtime_to_seconds("PT0S") == 0
+    assert leadtime_to_seconds("PT1M") == 0
+    assert leadtime_to_seconds("PT1H") == 3600
+    assert leadtime_to_seconds("PT1H30M") == 3600
+
+    assert leadtime_to_seconds("P1D") == 86400
+    assert leadtime_to_seconds("P1DT1H") == 90000
+    assert leadtime_to_seconds("P1DT1H30M") == 90000
+
+    assert leadtime_to_seconds("MT0S") == -0
+    assert leadtime_to_seconds("MT1H") == -3600
+    assert leadtime_to_seconds("MT1H30M") == -3600
+
+    assert leadtime_to_seconds("M1D") == -86400
+    assert leadtime_to_seconds("M1DT1H") == -90000
+    assert leadtime_to_seconds("M1DT1H30M") == -90000
+    assert leadtime_to_seconds("M1DT1H30M") == -90000
+    
+
```

Comment:
These numbers are pretty off, also there are only 13 entries for `leadtime_to_seconds` and 14 supplied values for `seconds_to_leadtime`. So the values are a little off. The numbers in the code accurately pass the tests. 

`leadtime.leadtime_to_seconds`

- "PT0S"
- **_"PT1M"_**
- "PT1H"
- **_"PT1H30M"_**

- "P1D"
- "P1DT1H"
- **_"P1DT1H30M"_**

- "MT0S"
- "MT1H"
- **_"MT1H30M"_**

- "M1D"
- "M1DT1H"
- **_"M1DT1H30M"_**


`leadtime.seconds_to_leadtime`
- 0
- 60
- 3600
- 5400

- 86400
- 90000
- 91800

- -0
- **_-60_**
- -3600
- -5400

- -86400
- -90000
- -91800

Reply:
`assert leadtime_to_seconds("PT1M") == 0`
should be
`assert leadtime_to_seconds("PT1M") == 60`

Reply:
`assert leadtime_to_seconds("PT1H30M") == 3600`
should be
`assert leadtime_to_seconds("PT1H30M") == 5400`

Reply:
`assert leadtime_to_seconds("P1DT1H30M") == 90000`
should be
`assert leadtime_to_seconds("P1DT1H30M") == 91800`

Reply:
`assert leadtime_to_seconds("MT1H30M") == -3600`
should be
`assert leadtime_to_seconds("MT1H30M") == -5400`

Reply:
`assert leadtime_to_seconds("M1DT1H30M") == -90000`
should be
`assert leadtime_to_seconds("M1DT1H30M") == -91800`

Reply:
It appears I accidentally left out the `MT1M` case.  That should evaluate to `-60`.  Perhaps the missing case caused mapping from leadtime to seconds to get messed up in the issue.  Sorry about that.  I'll go edit the issue and fix it.